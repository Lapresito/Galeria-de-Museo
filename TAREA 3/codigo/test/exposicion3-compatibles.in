crearExposicion 01/01/2002 01/06/2002 10 1
agregarAExposicion 5 0 1 2 3 4
imprimirExposicion
sonExposicionesCompatibles 01/01/2002 01/06/2002 10 2 5 0 1 2 3 4
sonExposicionesCompatibles 01/01/2002 01/06/2002 10 2 5 5 6 7 8 9
liberarExposicion
Fin