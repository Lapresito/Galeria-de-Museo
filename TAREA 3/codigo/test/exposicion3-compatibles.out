Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> Exposicion creada con exito.
2> Piezas agregadas a la exposicion de forma exitosa.
3> Exposicion #1 del 1/1/2002 al 1/6/2002
Piezas: 0 1 2 3 4 
4> Las exposiciones NO son compatibles.
5> Las exposiciones son compatibles.
6> Exposicion liberada con exito.
7> Fin.
